routers/api/v1/repo/branch.go

```diff
@@ -260,22 +260,29 @@ func ListBranches(ctx *context.APIContext) {
 	}
 
 	apiBranches := make([]*api.Branch, len(branches))
```

Comment:
```suggestion
	apiBranches := make([]*api.Branch, 0, len(branches))
```
so you wont have `""` in your list if you skip one

Reply:
https://github.com/go-gitea/gitea/pull/19547/commits/949c848f9db4278c6e7b7b8b3b801d58cb7dfb6d one step ahead of you 😄

Reply:
I still think we should add this patch too

Reply:
Why? It doesn't add any value after that commit.

Reply:
since it's a pointer I think it's ok ... - if it would be []string - I would prefere that (asuming go dont init it fully)

Reply:
Used it anyway, allows for shorter code.